tests/test_subscriber.py

```diff
@@ -193,19 +195,160 @@ def test_on_put_invalid(self, mock_invalid_param):
         self.assertTrue('invalid_param' in str(context.exception))
 
 
+@patch.object(Persister, 'create_indexes')
+@patch.object(Config, 'load_defaults')
+@patch('history.subscriber.persister.Messenger')
+@patch.object(Messenger, 'create_channel', return_value=None)
+@patch.object(Messenger, 'on', return_value=None)
+def test_persist_all_events(mock_on, mock_create_channel, mock_messenger, mock_config, mock_create_indexes):
+
+    from history.subscriber.persister import start_dojot_messenger
+    from history import conf
+
+    p = Persister()
+    p.create_indexes_for_notifications('admin')
+
+    mock_config.dojot = {
+        "management": {
+            "user": "dojot-management",
+            "tenant": "dojot-management"
+        },
+        "subjects": {
+            "tenancy": "dojot.tenancy",
+            "devices": "dojot.device-manager.device",
+            "device_data": "device-data"
+        }
+    }
+
+    # test persist all boolean valued events
+    conf.dojot_persist_notifications_only = False
```

Comment:
Are you sure that the value set in this line (224) influences something in the test?